cmake/projects/Qt/schemes/url_sha1_qt.cmake.in

```diff
@@ -215,6 +216,10 @@ if(BUILD_SHARED_LIBS)
   list(APPEND configure_opts "-shared")
 else()
   list(APPEND configure_opts "-static")
+  hunter_check_toolchain_definition(NAME "_DLL" DEFINED _hunter_vs_md)
```

Comment:
Not working. Some variables are not ready at this point. It's easier to call `hunter_check_toolchain_definition` in `hunter.cmake`.

To test you change locally:
* https://docs.hunter.sh/en/latest/creating-new/create/cmake.html#testing-locally